# size of the median filter, used for each sector
median_filter_size: 10
# how often data from sensors are evaluated and the bumper topic published
update_rate: 10
# number of sectors
n_horizontal_sectors: 8

lidar2d:
  # This is the length of the filter buffer, applied to 2D lidar scans when detecting
  # closest obstacle. A buffer of *lidar2d/filter_size* last measurements is kept
  # when sequentially going through the scan. If the maximal value from this buffer is
  # smaller than the minimal obstacle distance found in the current scan, the obstacle
  # distance is updated. This should filter out solitary false positives in the laser
  # scan (at least *lidar2d/filter_size* consecutive measurements have to be close
  # for an obstacle to be reported). For more details, see code (method
  # *find_obstacles_in_horizontal_sectors_robust*).
  filter_size: 5
